### DISCUSSION TITLE
Possible useRequestURL bug?

### DISCUSSION THREAD

**Discussion by timdiacon** [nuxt/nuxt#23076] [Questions]
I am using the useRequestURL in a component in a Nuxt project which I am running `yarn generate` on to produce a static site. It all works and I can deploy fine but I get the following error message in the console:

`Export "useRequestURL" of module "node_modules/nuxt/dist/app/composables/url.js" was reexported through module "node_modules/nuxt/dist/app/composables/index.js" while both modules are dependencies of each other and will end up in different chunks by current Rollup settings. This scenario is not well supported at the moment as it will produce a circular dependency between chunks and will likely lead to broken execution order.`

This seems like a bug but wanted to check if there is something I should be doing to avoid getting the error message?

**Comment by colearendt** [comment]
Has anyone had a solution/workaround to this problem? My error message warned about my importing `useRequestURL` from `#app` or `nuxt/app` (suggested removing the import line or importing directly from `url.js`). Removing the import did not work, though... and I was not able to craft an import that made things happy.

Anyone fortunate in crafting an import that works or a rollup config that avoids the issue? 

The additional detail I got in my error:
```
Either change the import in "pages/dev.vue?vue&type=script&setup=true&lang.ts" to point directly to the exporting module or reconfigure "output.manualChunks" to ensure these modules end up in the same chunk.
```

The issue showed up for me when bumping from `nuxt@3.6.1` to `nuxt@3.6.5` and associated deps. I.e. vite-builder from `3.6.5` to `3.7.4`, etc.
